src/pos.js

```diff
@@ -23,9 +26,73 @@ export class TransbankPOSWebSocket {
     async connect(socketIoUrl = "http://localhost:8090") {
         this.socket = io("http://localhost:8090")
         this.isConnected = true
+
+        this.socket.on("connect", () => {
+            this.isConnected = true;
+            this.checkAgentVersion();
+            this.checkSDKVersion();
```

Comment:
Maybe this check could be done outsite de on connect, but i'm not sure.

Reply:
It can be left out, as this now alerts you every time you connect with the agent.